build.gradle

```diff
@@ -1031,15 +1129,29 @@ project(':iceberg-spark3-runtime') {
     }
   }
 
+  ext {
+    prop = findProperty('spark31Runtime')
```

Comment:
If we need to have a separate module, then we should not enable it with a property. We should just add another runtime module. Otherwise, it gets very confusing that you can produce two artifacts with different compatibility from the same module.

Reply:
I removed the use of the property and added another runtime module instead.